### PR TITLE
TASK: Explain how to disable NodeTypes in documentation

### PR DESCRIPTION
**Pull request by DrillSergeant**
This adds documentation about how to disable NodeTypes. This was discussed
in https://github.com/neos/neos-development-collection/issues/1674

### CODE REVIEW

File: Neos.Neos/Documentation/CreatingASite/NodeTypes/DisableNodeTypes.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
+.. _disable-nodetypes:
+
+Disable NodeTypes
+===================
+
+To hide an existing NodeType (i.e. one that comes with Neos already) you have 2 options.
```
Comment (bwaidelich) [inline Neos.Neos/Documentation/CreatingASite/NodeTypes/DisableNodeTypes.rst]: "i.e." doesn't mean for example but more like "that means".
"e.g." or "for example" would be more correct I think

File: Neos.Neos/Documentation/CreatingASite/NodeTypes/DisableNodeTypes.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+Hide the NodeType from the user interface
+=========================================
+
+*NodeTypes.yaml*
+
+.. code-block:: yaml
+
+  Vendor.Site:YourContentElementName:
```
Comment (bwaidelich) [inline Neos.Neos/Documentation/CreatingASite/NodeTypes/DisableNodeTypes.rst]: The node type should be surrounded with single quotes

File: Neos.Neos/Documentation/CreatingASite/NodeTypes/DisableNodeTypes.rst [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+shown anymore in the dialog for adding nodes.
+
+Completely disallow the direct usage of a NodeType
+==================================================
+
+*NodeTypes.yaml*
+
+.. code-block:: yaml
+
+  Vendor.Site:YourContentElementName:
```
Comment (bwaidelich) [inline Neos.Neos/Documentation/CreatingASite/NodeTypes/DisableNodeTypes.rst]: single quotes

File: Neos.Neos/Documentation/CreatingASite/NodeTypes/DisableNodeTypes.rst [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    abstract: TRUE
+
+As abstract NodeTypes are not valid to be used directly this will hide the NodeType in the user interface AND
+additionally make all existing nodes of this type invalid. If you run a node:repair all existing nodes of this type will
```
Comment (bwaidelich) [inline Neos.Neos/Documentation/CreatingASite/NodeTypes/DisableNodeTypes.rst]: "node:repair" could be quoted with backticks to hightlight it as a command:
\`node:repair\`

**Comment by DrillSergeant** [comment]
Thank you @bwaidelich, I fixed those with a new commit

**Pull request merged**